cmd/otelcol/Dockerfile

```diff
@@ -53,6 +60,7 @@ COPY --from=otelcol --chown=999 /migratecheckpoint /
 COPY --from=otelcol --chown=999 /otel /etc/otel
 COPY --from=otelcol --chown=999 /otel/collector /etc/otel/collector
 COPY --from=agent-bundle --chown=999 /usr/lib/splunk-otel-collector /usr/lib/splunk-otel-collector
+COPY --from=jmx --chown=999 /tmp/opentelemetry-java-contrib-jmx-metrics.jar /opt/opentelemetry-java-contrib-jmx-metrics.jar
```

Comment:
This is the [default path](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/jmxreceiver#jar_path-default-optopentelemetry-java-contrib-jmx-metricsjar) for the receiver so users would have to otherwise set it manually. Do any issues come to mind or is there an existing reason to mount /opt that could interfere?

Reply:
Ah, makes sense.